src/ethereum/rlp.py

```diff
@@ -237,7 +237,7 @@ def _decode_to(cls: Type[T], raw_rlp: RLP) -> T:
             for t, raw_item in zip(cls.__args__, raw_rlp):  # type: ignore
                 args.append(_decode_to(t, raw_item))
             return tuple(args)  # type: ignore
-    elif cls == Union[Bytes0, Bytes20]:
+    elif cls == Union[Bytes0, Bytes20] or cls == Bytes20:
```

Comment:
This new line doesn't match the subsequent comment. How about something like:

```suggestion
    elif issubclass(cls, FixedBytes):
        ensure(isinstance(raw_rlp, Bytes), RLPDecodingError)
        ensure(len(raw_rlp) == cls.LENGTH, RLPDecodingError)
        return cls(raw_rlp)  # type: ignore
    elif cls == Union[Bytes0, Bytes20]:
```

Reply:
Updated